geomstats/geometry/quotient_metric.py

```diff
@@ -70,20 +49,20 @@ def inner_product(
         """
         if fiber_point is None:
             if base_point is not None:
-                fiber_point = self.fiber_bundle.lift(base_point)
+                fiber_point = self._space.lift(base_point)
```

Comment:
This is a bit puzzling as the `QuotientMetric` equips the base manifold, not the total space which is also the fiber bundle. So I think `_space` should be the base here, and we still need to access the fiber_bundle.

Reply:
You fully right here @nguigs! I've misunderstood this part: I thought the metric was to put in the `FiberBundle`, as it is done with the `PreShapeSpace`. (I also understand better your comments @ninamiolane.)

I can solve it easily for the case where the `QuotientMetric` is defined by the submersion (scenario 1 in your paper). I still need to think a bit more for scenario 2.

I'll change the code accordingly and the we iterate.